src/app/components/header/menu/menu.component.html

```diff
@@ -0,0 +1,39 @@
+<nav>
+  <ul [ngClass]="{ row: !isMobile, colum: isMobile }">
+    <li><a [routerLink]="this.navLinks.guest.home">Home</a></li>
+    <ng-container *ngIf="role === 'user'">
+      <li>
+        <a [routerLink]="this.navLinks.user.profile">Profile</a>
+      </li>
+      <li>
+        <a [routerLink]="this.navLinks.user.orders">My Orders</a>
+      </li>
+    </ng-container>
+    <ng-container *ngIf="role === 'admin'">
+      <li>
+        <a [routerLink]="this.navLinks.admin.profile">Profile</a>
+      </li>
+      <li>
+        <a [routerLink]="this.navLinks.admin.orders">Orders</a>
+      </li>
+    </ng-container>
+  </ul>
+</nav>
+<div class="buttons" [ngClass]="{ row: !isMobile, colum: isMobile }">
+  <ng-container *ngIf="role === 'guest'">
+    <button mat-flat-button [routerLink]="this.navLinks.guest.signup">
+      Sign Up
+    </button>
+    <button mat-flat-button [routerLink]="this.navLinks.guest.signin">
+      Sign In
+    </button>
+  </ng-container>
+  <ng-container *ngIf="role === 'user'">
+    <button mat-flat-button (click)="onLogout()">Logout</button>
+  </ng-container>
+  <ng-container *ngIf="role === 'admin'">
```

Comment:
We don't have 'admin' role, in our app it's 'manager'
![image](https://github.com/user-attachments/assets/c7af916f-d6ee-46d8-858e-d76cfee8b46b)

Reply:
Okay, I'll correct the role name everywhere to the current one before the merge.

Reply:
Done

Reply:
Also had this error)))